docs/extend/modular-resources/create/_index.md

```diff
@@ -478,6 +476,20 @@ For example:
 #!/bin/sh
 cd `dirname $0`
 
+# Be sure to use `exec` so that termination signals reach the python process,
+# or handle forwarding termination signals manually
+exec $PYTHON src/main.py $@
+```
+
+Make sure that you set up a Python virtual environment in the directory your module is in to compile your resource properly at execution if using the above script.
```

Comment:
Does this just require `pip3 install --target=folder_name viam-sdk` - if so might be easier to add that here. If not ignore me